ci/moose_utils.py

```diff
@@ -169,12 +169,12 @@ def unset_cargo_dependencies():
                 )
             _write_file(f"{PROJECT_ROOT}/Cargo.toml", replaced_moose)
 
-    # remove moosemeshnetapp dependency from telio-lana/Cargo.toml
+    # remove mooselibtelioapp dependency from telio-lana/Cargo.toml
```

Comment:
Maybe you know the reason why they renamed the tracker?

Reply:
No idea